lib/components/ui/popover/index.tsx

```diff
@@ -28,4 +28,33 @@ const PopoverContent = React.forwardRef<
 ));
 PopoverContent.displayName = PopoverPrimitive.Content.displayName;
 
-export { Popover, PopoverTrigger, PopoverContent };
+const ClickablePopover = ({
```

Comment:
If I'm understanding what this does correctly, when we open the ClickablePopover we'll wait 2 seconds and then close it. Is the use case clicking on something to show more info?